support-workers/cloud-formation/src/templates/state-machine.yaml

```diff
@@ -208,36 +208,36 @@ States:
     # These choices have a Next set to the steps generated by `supporterPlusProduct` below
     # we use isUS=true as currency is the only useful discriminator, but Int and US both use USD
     Choices:
-      {{> supporterPlusProductChoice currency="GBP" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_UK_MONTHLY_V2" }}
-      {{> supporterPlusProductChoice currency="EUR" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_EU_MONTHLY_V2" }}
-      {{> supporterPlusProductChoice currency="USD" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_INT_MONTHLY_V2" fulfilmentOptions="RestOfWorld" }}
-      {{> supporterPlusProductChoice currency="USD" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_US_MONTHLY_V2" fulfilmentOptions="Domestic" }}
-      {{> supporterPlusProductChoice currency="CAD" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_CA_MONTHLY_V2" }}
-      {{> supporterPlusProductChoice currency="NZD" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_NZ_MONTHLY_V2" }}
-      {{> supporterPlusProductChoice currency="AUD" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_AU_MONTHLY_V2" }}
-      {{> supporterPlusProductChoice currency="GBP" billingPeriod="Annual" promoCode="3TIER_WEEKLY_UK_ANNUAL_V2" }}
-      {{> supporterPlusProductChoice currency="EUR" billingPeriod="Annual" promoCode="3TIER_WEEKLY_EU_ANNUAL_V2" }}
-      {{> supporterPlusProductChoice currency="USD" billingPeriod="Annual" promoCode="3TIER_WEEKLY_INT_ANNUAL_V2" fulfilmentOptions="RestOfWorld" }}
-      {{> supporterPlusProductChoice currency="USD" billingPeriod="Annual" promoCode="3TIER_WEEKLY_US_ANNUAL_V2" fulfilmentOptions="Domestic" }}
-      {{> supporterPlusProductChoice currency="CAD" billingPeriod="Annual" promoCode="3TIER_WEEKLY_CA_ANNUAL_V2" }}
-      {{> supporterPlusProductChoice currency="NZD" billingPeriod="Annual" promoCode="3TIER_WEEKLY_NZ_ANNUAL_V2" }}
-      {{> supporterPlusProductChoice currency="AUD" billingPeriod="Annual" promoCode="3TIER_WEEKLY_AU_ANNUAL_V2" }}
+      {{> supporterPlusProductChoice currency="GBP" billingPeriod="Monthly" promoCode="3TIER_WEEKLY_UK_MONTHLY_V2" key="V2" }}
```

Comment:
Shame we could'nt use the promoCode name and had to use a key (80char limit I beleieve), hey ho!